src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -386,6 +410,7 @@ internal void Log(RpcEvent msg)
 
         internal void HandleWorkerChannelError(Exception exc)
         {
+            _workerInitTask.SetResult(false);
```

Comment:
Trying to ensure that _workerInitTask dependents don't hang

Reply:
PR: https://github.com/Azure/azure-functions-host/pull/4877 handles this